src/ui-kit/experimental/hierarchical/autocomplete/autocomplete.component.spec.ts

```diff
@@ -8,6 +8,7 @@ import { Observable } from 'rxjs';
 import { By } from '@angular/platform-browser';
 import 'rxjs/add/observable/of';
 
+
 fdescribe('SamHierarchicalAutocompleteComponent', () => {
```

Comment:
@thomastighe Please unfocus the test before merging

Reply:
Done. I commented out a test in another experimental control that was causing the build to fail